templates/docs/examples/patterns/logo-section/logo-section-dense.html

```diff
@@ -0,0 +1,28 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Logo section Dense{% endblock %}
```

Comment:
Small fix for our title convention

```suggestion
{% block title %}Logo section / Dense{% endblock %}
```